axonserver/src/main/java/io/axoniq/axonserver/plugin/DefaultPluginController.java

```diff
@@ -71,13 +73,24 @@ public List<PluginPropertyGroup> listProperties(PluginKey pluginKey, String cont
 
     @Override
     public void updateConfiguration(PluginKey pluginKey, String context,
-                                    Map<String, Map<String, Object>> properties) {
+                                    Map<String, Map<String, Object>> properties) throws ConfigurationValidationException {
         properties = PluginPropertyUtils.validateProperties(properties,
                                                             listProperties(pluginKey, Topology.DEFAULT_CONTEXT));
-        pluginContextManager.updateConfiguration(Topology.DEFAULT_CONTEXT,
-                                                 pluginKey.getSymbolicName(),
-                                                 pluginKey.getVersion(),
-                                                 properties);
+
+        Map<String, Iterable<ConfigurationError>> errors = configurationManager.errors(
+                pluginKey,
+                Topology.DEFAULT_CONTEXT,
+                properties
+        );
+
+        if (errors.isEmpty()){
+            pluginContextManager.updateConfiguration(Topology.DEFAULT_CONTEXT,
```

Comment:
again, it seams this should not be default context, but context that is passed via argument?